### PR TITLE
Simplify solution for ii_collections/n19Sum.kt

### PR DESCRIPTION
**Pull request by ranjeevmahtani**
I came up with this solution while going through the exercise. Wondering if it's a slightly simpler and more efficient one.

### CODE REVIEW

File: src/ii_collections/n19Sum.kt [hunk 1]
```diff
@@ -7,5 +7,5 @@ fun example6() {
 fun Customer.getTotalOrderPrice(): Double {
     // Return the sum of prices of all products that a customer has ordered.
     // Note: a customer may order the same product for several times.
-    return orders.flatMap { it.products }.map { it.price }.sum()
+    return orders.flatMap { it.products }.sumByDouble { it.price }
```
Comment (piotrek1543) [inline src/ii_collections/n19Sum.kt]: Wow, I haven't thought there could be easier solution for this. Thanks!

Comment (ranjeevmahtani) [inline src/ii_collections/n19Sum.kt]: No worries. Is this good to be merged in as the preferred solution?

Comment (piotrek1543) [inline src/ii_collections/n19Sum.kt]: @ranjeevmahtani I am just contributer like you, so I won't decide. I can only say that your solution is really interesting, but as `kotlin-koans` is dedicated for beginners, this solution might be a bit tough to guess for guys who don't know kotlin syntax sugar as good as you.

Comment (ranjeevmahtani) [inline src/ii_collections/n19Sum.kt]: Ah I see! Sorry, I misunderstood your role :)
I just got really excited about potentially contributing something to the community here.

Comment (piotrek1543) [inline src/ii_collections/n19Sum.kt]: @ranjeevmahtani I understand, don't miss other opportunities: https://github.com/trending/kotlin

**Pull request merged**